src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelEvents.cs

```diff
@@ -77,6 +77,24 @@ public virtual void OnRequestAboutWindow(DynamoViewModel vm)
             }
         }
 
+        public event RequestShowGalleryHandler RequestShowGallery;
+        public virtual void OnRequestShowGallery()
+        {
+            if (RequestShowGallery != null)
+            {
+                RequestShowGallery();
+            }
+        }
+
+        public event RequestCloseGalleryHandler RequestCloseGallery;
+        public virtual void OnRequestCloseGallery()
+        {
+            if(RequestCloseGallery != null)
+            {
+                RequestCloseGallery();
+            }
+        }
+
```

Comment:
Can we merge these two into one? Like `OnRequestShowHideGallery(bool showGallery)`? And so we will only need one `RequestShowHideGalleryHandler`.